### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by PowerPress** [ervanalb/detour#14]
This is an awesome project any plans to add any other features or enhancements?

**Comment by zbanks** [comment]
Hey @PowerPress 

We haven't been doing a lot with this project, but recently I've been re-writing the software stack in C (the python code is very "proof of concept" quality...). I really want to get it running on cheap routers like this, which contain _exactly_ the major hardware required (2 ethernet ports + wifi + USB + GPIO)

https://www.amazon.com/GL-MT300N-Standard-pre-installed-Repeater-Tethering/dp/B01AL7P1FU/ref=sr_1_6?ie=UTF8&qid=1475111562&sr=8-6&keywords=travel+router

There's some more changes I'd also like to make to the tap board, but I want to get the software more polished first :smile: 

---

What parts of the project are you interested in? It's worth noting the software side can be demonstrated without any special hardware -- with a few modifications it could even work with the more common ARP spoofing MITM attack.

**Comment by StephenGenusa** [comment]
Incredibly awesome board guys. I'm interested in using the board to experiment with modifying IP streams but I can see where it has a variety of uses.

I'm looking at getting some PCB's made through the Seeed Studio PCB service. Before I do this Eric, I wanted to check where you were in possible development of other hardware. I hate to pay somewhere in the ballpark of $250 per board, if something else is in the pipe. Also, is there any problem selling off the extra boards since there is a minimum quantity on the order?

The CDBA120-G is not available. I think the CDBA140-G is a compatible substitute. Just wanted to verify that if possible.

I would appreciate any input before I go head first ;-).

**Comment by zbanks** [comment]
Thanks @StephenGenusa , we don't have any plans for the near future. We also don't have a ton of spare boards but I'll let @ervanalb answer that. CDBA140-G looks good to me as a replacement part.

Also as I told PowerPress, you can play with the software side without manufacturing any hardware if you want to get your feet wet first.

**Comment by ervanalb-vs** [comment]
Nah, there is no new board in the near future. I think I gave away all of the extra PCB at DEF CON. I would second @zbanks suggestion to bring up the software and play with that. The PCB is optional unless you are truly looking to covertly splice live Ethernet cable. If you are fine with unplugging / plugging Ethernet cables, you don't need the PCB.

**Comment by StephenGenusa** [comment]
Thank you Zach and Eric 👍
